strawberry/annotation.py

```diff
@@ -179,6 +181,13 @@ def create_union(self, evaled_type) -> StrawberryUnion:
         union = StrawberryUnion(
             type_annotations=tuple(StrawberryAnnotation(type_) for type_ in types),
         )
+
+        for arg in args:
+            if isinstance(arg, StrawberryUnion):
```

Comment:
Considering duplicate types in `Annotated` are removed, can this loop be replaced by a lookup of the single StrawberryUnion arg?
![image](https://github.com/strawberry-graphql/strawberry/assets/11272260/f8cd1d73-061a-457c-b358-20d45991afbf)
[PEP-593](https://peps.python.org/pep-0593/)